src/validators/union.rs

```diff
@@ -407,7 +455,7 @@ impl TaggedUnionValidator {
     fn find_call_validator<'s, 'data>(
         &'s self,
         py: Python<'data>,
-        tag: Cow<str>,
+        tag: Cow<ChoiceKey>,
```

Comment:
I would guess this can just be a ref? - the reference can live long enough in all cases above.

Reply:
yes thanks for the suggestion